shared-data/python/opentrons_shared_data/protocol/models/protocol_schema_v6.py

```diff
@@ -0,0 +1,151 @@
+from pydantic import BaseModel, Field
+from typing import List, Optional, Dict
+from typing_extensions import Literal
+from enum import Enum
+from opentrons_shared_data.labware.labware_definition import LabwareDefinition
+
+
+class CommandAnnotation(BaseModel):
+    commandIds: List[str]
+    annotationType: str
+
+
+class CornerOffsetFromSlot(BaseModel):
+    x: int
+    y: int
+    z: int
```

Comment:
I wonder if we can find a better name for this. Also, the params need to be floats, not ints

```suggestion
class OffsetVector(BaseModel):
    x: float
    y: float
    z: float
```